hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Hangman.Game do
```

Comment:
This is beautiful. Thank you!

I love the structure _and_ the layout. It is clear to read.

## +5 for making me happy :)